src/renderers/common/Renderer.js

```diff
@@ -344,6 +346,20 @@ class Renderer {
 
 	}
 
+	setMRT( mrt ) {
```

Comment:
When I understand the code correctly this makes it possible that a pass can dynamically use `setMRT()` (e.g. in `updateBefore()`) to configure the renderer for its beauty/depth/normal pass and then restores the setting to the previous value, right?

Reply:
Once again I'm amazed with how less changes something like this can be integrated in the renderer/material system 😮!

Reply:
Almost there!

```js
postProcessing = new THREE.PostProcessing( renderer );
postProcessing.outputColorTransform = false;
postProcessing.setMRT( mrt( {
	'output': output,
	'normal': transformedNormalWorld.directionToColor()
} ) );
postProcessing.outputNode = mix( scenePass.renderOutput(), scenePass.getTextureNode( 'normal' ), step( 0.5, uv().x ) );
```

![image](https://github.com/mrdoob/three.js/assets/502810/29e4a2c2-40b9-4225-9aad-6782abb2dc99)

Reply:
It is now possible to blur the emissive and add it on the post-processing in a one-line :)

```js
postProcessing.outputNode = scenePass.add( scenePass.getTextureNode( 'emissive' ).gaussianBlur( 3, 10 ) ).renderOutput();
```

![image](https://github.com/mrdoob/three.js/assets/502810/5bf3d32d-620a-47ac-953e-d0dc793ea044)

Reply:
```js
postProcessing = new THREE.PostProcessing( renderer );
postProcessing.outputColorTransform = false;
postProcessing.setMRT( mrt( {
	'output': output,
	'normal': transformedNormalWorld.directionToColor()
} ) );
```
What is the reason for configuring MRT on `PostProcessing` level? I would expect `setMRT()` is called on passes instead. Like:
```js
const scenePass = pass( scene, camera );
scenePass.setMRT( mrt( {
	'output': output,
	'normal': transformedNormalWorld.directionToColor()
} ) );
```
How would `PostProcessing.setMRT()` be used if an effect later in the pass chain internally uses MRT to produce its output? Meaning you would more than one pass with different MRT setup.

Reply:
> It is now possible to blur the emissive and add it on the post-processing in a one-line :)

Damn, this is essentially a one-line bloom effect!^^

Reply:
I added to pass() too, in PostProcessing should apply in all passes automatically if it's not defined in pass()

Reply:
I think that we could remove setMRT of PostProcessing without problems too, maybe it's force user create better approach.

Reply:
I would prefer to remove for now. We can add it later back if it turns out we require it.

Reply:
I changed it, I think it looks better.